src/pyfx/view/json_lib/node_factory.py

```diff
@@ -32,7 +33,26 @@ def create_node(self, key, value, **kwargs):
         """
         node_impl = self._node_map[type(value)]
 
-        if issubclass(node_impl, JSONCompositeNode):
-            # a complex object should inherited JSONCompositeNode
-            return node_impl(key, value, self, **kwargs)
-        return node_impl(key, value, **kwargs)
+        try:
+            if issubclass(node_impl, JSONCompositeNode):
+                # a complex object should inherited JSONCompositeNode
+                    return node_impl(key, value, self, **kwargs)
+            return node_impl(key, value, **kwargs)
+        except Exception as error:
+            # Some non-json datatypes can throw errors in specific situations,
+            # and when this happens we should fall back to GenericNode instead of crashing.
+            # One example of this being useful is when displaying torch.Tensor objects:
+            # torch.Tensors have a __len__, because they can contain many numbers.
+            # However, the individual numbers in these tensors are also of type torch.Tensor,
+            # and torch throws an error when you try to get len(x) where x is a single number of type torch.Tensor.
+            # TLDR: This fallback lets us display torch.Tensor
+
+            #During development of pyfx, it might help to uncomment the following warning.
+            #During deployment, however, it's quite annoying and best left silent.
+            # logger.warning(
```

Comment:
does using `logger.debug(...)` helps reduce the warning? the warning messages shown in the UI are also unexpected to me.

Reply:
If you're asking whether uncommenting the logger.warning and replacing it with logger.debug will make the errors invisible, unfortunately the answer is no :\ Sometimes they appear and dissapear again quickly, but they shouldn't appear at all during usage

Reply:
Yes, I understand. then let's comment it for now, and I'll see if I can fix it.